### ISSUE TITLE
values from localstorage

### ISSUE THREAD

**Issue by zefanja** [cdscacth/AQI#8]
If a value from localstorage is older than e.g. 2 hours than we shouldn't take it into account for the avarage AQI calculation. Otherwise the resulting AQI can be bit misleading.

What do you think?

**Comment by aPinat** [comment]
Yeah, true... Although it's quite unrealistic, what we don't get any new values for two hours; plus the timestamp is still shown...

But it could happen, when the website is still open after a devices awakes from standby etc...
But, I'll think about it, when the structural changes to the new calculation method of using only CDSC values is done...
